src/molecules/BottomSheet/BottomSheet.native.js

```diff
@@ -0,0 +1,117 @@
+import React, { forwardRef } from 'react';
+import { Dimensions } from 'react-native';
+import styled, { useTheme } from 'styled-components/native';
+import { Modalize as RNModalize } from 'react-native-modalize';
+import PropTypes from 'prop-types';
+import Flex from '../../atoms/Flex';
+import Space from '../../atoms/Space';
+import Size from '../../atoms/Size';
+import Position from '../../atoms/Position';
+import View from '../../atoms/View';
+import Divider from '../../atoms/Divider';
+
+const screenHeight = Dimensions.get('window').height;
+const DEFAULT_SNAP_POINT = screenHeight * 0.4; // 40% of screen height
+
+const styles = {
+  overlayStyle: ({ theme }) => {
+    return {
+      backgroundColor: theme.colors.shade[950],
+    };
+  },
+  childrenStyle: ({ theme }) => {
+    return {
+      backgroundColor: theme.colors.background[200],
+    };
+  },
+};
+
+const HeaderContainer = styled(View)`
+  background-color: ${(props) => props.theme.colors.background[200]};
+  border-top-right-radius: 8px;
+  border-top-left-radius: 8px;
+  box-shadow: ${(props) => `0px -4px 15px ${props.theme.colors.primary[930]}`};
+`;
+
+const BottomSheetDragBar = styled(View)`
+  background-color: ${(props) => props.theme.colors.shade[920]};
+  border-radius: 4px;
+`;
+
+const BottomSheet = forwardRef(
+  (
+    {
+      snapPoint = DEFAULT_SNAP_POINT,
+      children = null,
+      HeaderComponent = null,
+      FooterComponent = null,
```

Comment:
Can't it be undefined by default?

```suggestion
      children,
      HeaderComponent,
      FooterComponent,
```